Packs/ShiftManagement/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,6 @@
+## [Unreleased]
+- Add the GetShiftsPerUser automation
+
+### Scripts
+- __GetOnCallHoursPerUser__
+%%UPDATE_RN%%
```

Comment:
fill this

Reply:
Can you elaborate a bit what to do there? it's pretty cryptic...

Reply:
just add you RN here

Reply:
I did on top, no? What is `%%UPDATE_RN%%`? sorry for not understanding... is there a doc you can just forward me?

Reply:
i think that you should just fill it instead of the %% %% and not on top, @amshamah419 @dantavori correct me if i am wrong ( New format so I'm not too sure myself)

Reply:
@agamm See suggested changes

Reply:
@amshamah419 I committed your changes, is it good now?